README.md

```diff
@@ -75,7 +75,7 @@ to the correct version of `unixodbc`.
 
 ```shell
 nix run .#build
-./build/release/duckdb
+./build/release/duckdb -unsigned
```

Comment:
not required but I thought it was relevant as we are going to load the unsigned extension later on